sdmetrics/reports/utils.py

```diff
@@ -86,7 +86,8 @@ def make_discrete_column_plot(real_column, synthetic_column, sdtype):
     synthetic_data['Data'] = 'Synthetic'
 
     missing_data_real = round((real_column.isna().sum() / len(real_column)) * 100, 2)
-    missing_data_synthetic = round((synthetic_column.isna().sum() / len(synthetic_column)), 2)
+    missing_data_synthetic = round(
```

Comment:
I think one way we can avoid bugs like this in the future is by pulling all the logic into on common placce. I think it makes sense to have a private static method `_get_missing_percentage` that takes in the column and returns the percentage and then we can just call it on lines 88, 89, 157 and 158

Reply:
Yes, it's a good idea, I can do this